Extensions/Terraform/Src/Tasks/TerraformInstaller/src/terraform-installer.ts

```diff
@@ -0,0 +1,92 @@
+import tasks = require('azure-pipelines-task-lib/task');
+import tools = require('azure-pipelines-tool-lib/tool');
+import path = require('path');
+import os = require('os');
+import fs = require('fs');
+
+const uuidV4 = require('uuid/v4');
+const terraformToolName = "terraform";
+const isWindows = os.type().match(/^Win/);
+
+export async function downloadTerraform(inputVersion: string): Promise<string> {
```

Comment:
can you test in both windows and Linux agents?

Reply:
I have tested the tasks on both Windows and Linux agents.